_layouts/page.html

```diff
@@ -1,12 +0,0 @@
----
-layout: default
----
-<article class="post">
-
-  <h2 class="post-title">{{ page.title | escape }}</h2>
-
-  <div class="post-content">
-    {{ content }}
-  </div>
-
-</article>
```

Comment:
We need to keep this layout, as it is the generic layout for all pages that will come in the future that don't need a specialized layout. That being said, you could use the page layout for the about page, and use about.md to include the content includes:

```
---
layout: about
title: About
permalink: /about/
menu: true
---
{% include homegrown-software.html %}
{% include no-queens-no-keepers.html %}
{% include bringing-it-all-back.html %}